### PR TITLE
[RHDX-93] Update 'More like this' section to use new styles

### PR DESCRIPTION
**Pull request by staceymosier**
### JIRA Issue Link
[RHDX-93](https://projects.engineering.redhat.com/browse/RHDX-93)

Affects Video Resource content type in Full view only:
- https://localhost/videos/youtube/FC3jSKgTNK0/
- https://localhost/videos/youtube/Qcys7fKSzB0/
- https://localhost/videos/youtube/tI2l8yThKK4/
- https://localhost/videos/youtube/mhwvDeDzSSA/

PR environment links:
- https://developer-preview-3448.ext.us-west.dc.preprod.paas.redhat.com/videos/youtube/FC3jSKgTNK0/
- https://developer-preview-3448.ext.us-west.dc.preprod.paas.redhat.com/videos/youtube/Qcys7fKSzB0/
- https://developer-preview-3448.ext.us-west.dc.preprod.paas.redhat.com/videos/youtube/tI2l8yThKK4/
- https://developer-preview-3448.ext.us-west.dc.preprod.paas.redhat.com/videos/youtube/mhwvDeDzSSA/

### Verification Process
View any **Video Resource** and verify the More like this displays at the end of the content, but before the Disqus comments, and renders correctly in the card grid format. 

The **target product** shown in the 3rd example is reported as a [separate issue](https://projects.engineering.redhat.com/browse/RHDX-97). 
The **related links** show in the 4th example is reported as a [separate issue](https://projects.engineering.redhat.com/browse/RHDX-96)

Caveats:   The **related content** field that previously rendered within the 'more like this' list has been removed,  because:
1. It has been used only twice and the content renders in full display
2. The field accepts any content type, and not all content types are supported as cards at this time.

See [example](https://developers.redhat.com/videos/youtube/mhwvDeDzSSA/), "Helloworld Microservices Introduction" will no longer display after this is approved.

### CODE REVIEW

**Comment by jordanpagewhite** [comment]
rebuild this please

File: _docker/drupal/drupal-filesystem/web/themes/custom/rhdp2/js/related-content.js [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+        $($parent).removeClass('loading');
+      }
+    });
+
+    // $($parent).html(html.join(''));
+    $(html.join('')).appendTo($parent);
+    var $clampItems = $('.line-clamp-2', $parent);
+    $clampItems.each(function () {
+      var $tmpItem = $(this);
+      $clamp($tmpItem.get(0), { clamp: 2, useNativeClamp: true });
```
[the remark this answers is not in the document]
Comment (jordanpagewhite) [reply _docker/drupal/drupal-filesystem/web/themes/custom/rhdp2/js/related-content.js]: Yeah, I see how this is working now after reviewing it in the PR environment. This is working as intended.

[the remark this answers is not in the document]
Comment (staceymosier) [reply _docker/drupal/drupal-filesystem/web/themes/custom/rhdp2/js/related-content.js]: I tried to leave the JS as written,  I only changed the reference to the $parent in the various places it was written out.  Also making it a Drupal behavior.

File: _docker/drupal/drupal-filesystem/web/themes/custom/rhdp2/templates/node/node--video-resource--full.html.twig [hunk 2]
```diff
@@ -74,18 +74,17 @@
...
-  <section class="video-page related-content">
-    <div class="pf-l-grid">
-      <div id ="video-related-cont" class="pf-l-grid__item pf-m-12-col" data-tags="{{ video_tags|json_encode|replace('\"', '\'') }}">
-        <h4>MORE LIKE THIS</h4>
-        {{ content.field_related_content }}
-        <div class="video-related-content video-related-content-list"></div>
-      </div>
+  <section class="video-page rhd-c-card-grid">
+    <div class="pf-l-flex" >
+      <h2 class="pf-c-title">More Like This</h4>
```
Comment (mindreeper2420) [inline _docker/drupal/drupal-filesystem/web/themes/custom/rhdp2/templates/node/node--video-resource--full.html.twig]: This is the line where the properties do not match. Everything else looks great!

Comment (staceymosier) [reply _docker/drupal/drupal-filesystem/web/themes/custom/rhdp2/templates/node/node--video-resource--full.html.twig]: I pushed a change for this.  Thank you for catching that.

**Comment by staceymosier** [comment]
@jordanpagewhite   Not sure if this is related to changes or local env.

I am getting a JS error, Uncaught ReferenceError: sendCustomEvent is not defined,
from [5-sso.js](https://github.com/redhat-developer/developers.redhat.com/blob/f6f9fdb1c01c3ac3e511bd2445db40bb8465b503/_docker/drupal/drupal-filesystem/web/themes/custom/rhdp2/rhd-frontend/src/scripts/js/rhd/5-sso.js#L120)

**Pull request merged**